knowledge_graphs/hardware_optimized_i7_13700k.py

```
@@ -0,0 +1,407 @@
+Hardware-optimized configuration for Intel i7-13700K + 32GB DDR5 + RTX 4070 12GB VRAM.
+
+Based on concrete research and Intel/Neo4j official documentation for optimal database workload performance.
+
+Hardware Specifications (researched):
+- Intel i7-13700K: 16 cores (8 P-cores + 8 E-cores), 24 threads, up to 5.4GHz boost
+- 30MB L3 cache, 24MB L2 cache
+- TDP: 125W base, 253W max turbo power
+- Memory: 32GB DDR5 (optimal speeds: DDR5-6400 to DDR5-7200)
+- GPU: RTX 4070 12GB VRAM
+"""
+import os
```

Ruff (0.12.2)

`os` imported but unused

Remove unused import: `os`

(F401)